addons/misc/functions/fnc_strongpoint.sqf

```diff
@@ -0,0 +1,124 @@
+#include "script_component.hpp"
+/*/////////////////////////////////////////////////
+Author: Landric
+               
+File: fn_strongpoint.sqf
+Parameters: _dialogResult, _in
+Return: none
+
+*///////////////////////////////////////////////
+params ["_dialogResult","_in"];
+
+_in params [["_pos",[0,0,0],[[]],3], "_unit"];
+
+private _params = ["_composition", "_radius","_desiredStrongpointCount","_fill","_sandbags","_patrols"];
+if(!isNull _unit) then { _params = _params - ["_composition"]; };
+if(!EGVAR(main,zeiLoaded)) then { _params = _params - ["_sandbags"]; };
+
+_dialogResult params _params;
```

Comment:
should be commented out when done testing ;-)

Reply:
Hopefully by the time I'm finished, that line will be necessary :D